xls/contrib/mlir/IR/xls_ops.cc

```diff
@@ -390,6 +390,28 @@ Type getI1TypeOf(Type type) {
 
 namespace mlir::xls {
 
+LogicalResult LiteralOp::verifyRegions() {
+  Block& b = getInitializerBlock();
+  // Verify return op & typing:
+  auto ret = cast<YieldOp>(b.getTerminator());
```

Comment:
You shouldn't assume the terminator is a yield in a verifier (verify it, don't assert it).